e2e/test_module.py

```diff
@@ -13,15 +14,18 @@ def db():
 class TestConstants:
     ABSOLUTE_TOLERANCE = 1e-3
 
+    CLEANUP = "cleanup"
     EXCEPTION = "exception"
     INPUT_FILE = "input.cyp"
     OUTPUT = "output"
+    ONLINE_TEST = "_test"
+    SETUP = "setup"
```

Comment:
Similar note for this const name as below, I think we should set name that explains where are we using it
ONLINE_TEST_SETUP_KEYWORD 
or
ONLINE_TEST_E2E_SETUP

Reply:
Totally agreed!